packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '../../../lib/constants'
 
 let rateLimitedUntil = 0
-let memoryCache: LRUCache<string, CacheHandlerValue> | undefined
+export let memoryCache: LRUCache<string, CacheHandlerValue> | undefined
```

Comment:
is this going to be imported somewhere?

Reply:
Oh not anymore, leftover from previous approach